storage_image/static/src/js/image_handle.js

```diff
@@ -22,6 +32,7 @@ odoo.define("storage_image.image_handle", function(require) {
                     e.stopPropagation();
                     self.upload_images(e.originalEvent.dataTransfer.files);
                 });
+                self.events_added = true;
```

Comment:
No ?

```suggestion
                self.dnd_events_added = true;
```

Reply:
Yes, I wanted to see who would see this. Congrats @rousseldenis